samples/java_layout/src/main/java/io/customer/android/sample/java_layout/sdk/CustomerIORepository.java

```diff
@@ -50,6 +51,19 @@ public void initializeSdk(SampleApplication application) {
 
         // Finally, build to finish initializing the SDK.
         builder.build();
+
+        // TODO: Remove old builder and use new builder to initialize the SDK
+        // New method to initialize CustomerIO
+        // The new method should be called after the old method till the old method is removed
+        // This is because the push and in-app modules are still using properties only initialized in the old method
```

Comment:
Good comment. 

Are we tracking `TODO`s anywhere in our tickets? 

Once we merge this, I am curious how we keep ourselves accountable to following up to tasks like this and also being able to measure the amount of effort involved in each `TODO` we create in the code. 

I greatly appreciate the effort in keeping the CI passing, including sample apps building. Seems worth it to me.

Reply:
Great call. I've created a [ticket to address these TODOs](https://linear.app/customerio/issue/MBL-297/resolve-all-pending-todos-for-milestone-1) before we complete Milestone 1.

Regarding the efforts, I think it's challenging to set rules for it, but I've been trying to ensure that nothing breaks, all CI checks pass, and adding TODOs only when absolutely necessary.

Most of the TODOs so far involve straightforward code removal and renaming actions. I think we just need to be mindful of what we categorize as a TODO. If we notice that something added as a TODO requires a larger effort, we should highlight it and create a separate ticket for it.